application/src/main/java/uk/nhs/hee/tis/revalidation/validator/TraineeRecommendationRecordDTOValidator.java

```diff
@@ -21,18 +21,29 @@
 
 package uk.nhs.hee.tis.revalidation.validator;
 
+import static java.lang.String.format;
 import static java.time.LocalDate.now;
 
+import java.time.LocalDate;
 import java.time.temporal.ChronoUnit;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
 import org.springframework.util.StringUtils;
 import org.springframework.validation.Errors;
 import org.springframework.validation.Validator;
 import uk.nhs.hee.tis.revalidation.dto.TraineeRecommendationRecordDto;
 import uk.nhs.hee.tis.revalidation.entity.RecommendationType;
+import uk.nhs.hee.tis.revalidation.exception.RecommendationException;
+import uk.nhs.hee.tis.revalidation.repository.DoctorsForDBRepository;
 
+@Component
 public class TraineeRecommendationRecordDTOValidator implements Validator {
 
   public static final String INSUFFICIENT_EVIDENCE = "1";
+  private static final String DOCTOR_NOT_FOUND_MESSAGE = "Doctor %s does not exist!";
+
+  @Autowired
```

Comment:
The outcome of the discussion was to use Constructor injection where possible.  Is there a reason you've gone back to field injection?

Reply:
Sorry my bad. I should have used constructor injection. Changed now. Thanks.

Reply:
Sorry my rollback got rid of the constructor injection